pom.xml

```diff
@@ -2043,6 +2043,9 @@
         <weld3.version>3.0.0.Final</weld3.version>
         <xerces.version>2.11.0</xerces.version>
         <yasson.version>1.0.3</yasson.version>
+        <helidon.version>1.0.2</helidon.version>
+        <microprofile-api.version>1.2.1</microprofile-api.version>
```

Comment:
We may end up supporting different version for config and rest client. Please adjust the property name by "config".